src/language/completionItemProvider.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ColumnDefinition,
+  getStandardSQLCompletionProvider,
+  LanguageCompletionProvider,
+  TableDefinition,
+  TableIdentifier,
+  // getStandardSQLCompletionProvider,
```

Comment:
nit: commented code

Reply:
also I am geting several errors from this file

Reply:
What kind of errors? If you've previously used a locally linked version of grafana/experimental, you may have to make a hard reinstall. `yarn install --force && yarn`

Reply:
the same than in Drone: https://drone.grafana.net/grafana/redshift-datasource/412/1/5

Reply:
ah that's right - need to merge [this](https://github.com/grafana/grafana-experimental/pull/46) PR first :|

Reply:
Think the link above is pointing to another PR. Anyway, should be working fine now!